Intacct.SDK.Tests/Functions/AccountsReceivable/ArAdvanceUpdateTest.cs

```diff
@@ -0,0 +1,166 @@
+﻿/*
+ * Copyright 2020 Sage Intacct, Inc.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not
+ * use this file except in compliance with the License. You may obtain a copy 
+ * of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * or in the "LICENSE" file accompanying this file. This file is distributed on 
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either 
+ * express or implied. See the License for the specific language governing 
+ * permissions and limitations under the License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Xml;
+using Intacct.SDK.Functions.AccountsReceivable;
+using Intacct.SDK.Functions;
+using Intacct.SDK.Tests.Xml;
+using Intacct.SDK.Xml;
+using Xunit;
+
+namespace Intacct.SDK.Tests.Functions.AccountsReceivable
+{
+    public class ArAdvanceUpdateTest : XmlObjectTestHelper
+    {
+        [Fact]
+        public void GetXmlTest()
+        {
+            string expected = @"<?xml version=""1.0"" encoding=""utf-8""?>
+<function controlid=""unittest"">
+    <update>
+        <ARADVANCE>
+            <RECORDNO>123</RECORDNO>
+            <CUSTOMERID>CUSTOMER1</CUSTOMERID>
+            <PAYMENTMETHOD>Cash</PAYMENTMETHOD>
+            <PAYMENTDATE>06/30/2015</PAYMENTDATE>
+            <RECEIPTDATE>06/30/2015</RECEIPTDATE>
+            <ARADVANCEITEMS>
+                <ARADVANCEITEM>
+                    <TRX_AMOUNT>76343.43</TRX_AMOUNT>
+                </ARADVANCEITEM>
+            </ARADVANCEITEMS>
+        </ARADVANCE>
+    </update>
+</function>";
```

Comment:
please fix the indentation here and in all other places for similar construct

Reply:
what's wrong with the indentation?